### ISSUE TITLE
Request: Allow for downloading Community Posts (Images, Gifs)

### ISSUE THREAD

**Issue by hi-im-karma** [arichornlover/uYouEnhanced#453]
It would be awesome if we were able to download community posts in their highest quality and resolution.
Please consider implementing this.

![image](https://github.com/arichornlover/uYouEnhanced/assets/73917917/118b5d87-4591-4757-bd97-e597fff50f5b)

**Comment by KarmaBeast** [comment]
> I think the feature already exists in uYou. Try tapping and holding an image of a community post.

You're right. I wasn't aware of this long tap on image. **However I did test it on community posts that are GIFs, and it doesn't save them as GIFs but as static images**

So this uyou can only save image community posts, but not gif community posts. Can we add this functionality seperately?
